src/main/java/org/junit/runners/model/FrameworkMethod.java

```diff
@@ -149,6 +149,11 @@ public boolean isShadowedBy(FrameworkMethod other) {
     }
 
     @Override
+    boolean isBridgeMethod() {
```

Comment:
...and JUnit 4 does not provide [support](https://github.com/junit-team/junit5/tree/master/junit-platform-commons/src/main/java/org/junit/platform/commons/support) tools like JUnit 5 now does.